src/foam/core/JSON.js

```diff
@@ -155,6 +155,11 @@ foam.CLASS({
     },
     {
       class: 'Boolean',
+      name: 'outputOwnPropertiesOnly',
+      value: true
```

Comment:
This is probably the wrong default value since it changes the behaviour for others.